node_cli/core/node.py

```diff
@@ -294,13 +259,16 @@ def update(env_filepath: str, pull_config_for_schain: str, unsafe_ok: bool = Fal
         error_msg = 'Cannot update safely'
         error_exit(error_msg, exit_code=CLIExitCodes.UNSAFE_UPDATE)
 
+    prev_version = get_meta_info().version
+    logger.info('HERE %s %s', __version__, prev_version)
+    if (__version__ == 'test' or __version__.startswith('2.6')) and prev_version == '2.5.0':
```

Comment:
will it work ok if `migrate` will be executed multiple times? (let's say for 2.6.1, 2.6.2, etc)